index.bs

```diff
@@ -5822,6 +5822,12 @@ Since certificates are sizable relative to the storage capabilities of typical a
 
 Note: In order to interoperate, user agents storing large blobs on authenticators using [[!FIDO-CTAP]] are expected to use the provisions detailed in that specification for storing [=large, per-credential blobs=].
 
+Note: [[!FIDO-CTAP]] authenticators only support this extension for [=discoverable credentials=],
+and might return an incompatibility error unless
+<code>{{PublicKeyCredentialCreationOptions/authenticatorSelection}}.{{AuthenticatorSelectionCriteria/residentKey}}</code>
+is set to {{ResidentKeyRequirement/preferred}} or {{ResidentKeyRequirement/required}}.
+This is not generally a requirement for all kinds of authenticators.
```

Comment:
```suggestion
This is not a general requirement for all kinds of authenticators.
```

Reply:
Yeah, I went back and forth a lot on that one. :laughing: Thanks!